src/sidebars/search/search-compact.less

```diff
@@ -24,6 +24,18 @@
         }
       }
     }
+
+    &.search {
+      .search-result {
+        &.selected.dark {
```

Comment:
Same comment! `&.search .search-result.selected.dark` can still be the top line here.